rfcs/0001-AMP_pass.md

```diff
@@ -0,0 +1,137 @@
+- Feature Name: Automatic Mixed Precision Pass
+- Start Date: 2021-06-08 
+- RFC PR: TODO
+- GitHub Issue: TODO
```

Comment:
As this RFC is guaranteed to be merged and the feature must be landed, It should be fine to open a tracking issue now and update the link here.

Reply:
I found that the issue has been opened, so please just update the link here.